clarifai/client/app.py

```diff
@@ -288,6 +290,87 @@ def list_concepts(self, page_no: int = None,
       concept_info['id'] = concept_info.pop('concept_id')
       yield Concept(**concept_info)
 
+  def _rich_tree_creation_for_concept_relations(self, relations_dict: Dict[str, Any]) -> None:
+    """Print all the concept relations of the app in rich tree format.
+
+    Args:
+        relations_dict (dict): A dict of concept relations info.
+    """
+    for parent, children in relations_dict.items():
+      tree = Tree(parent)
+      for child in children:
+        tree.add(child)
+      rprint(tree)
```

Comment:
Can we move this function to logging utils, https://github.com/Clarifai/clarifai-python/blob/master/clarifai/utils/logging.py

Reply:
Done